mne/viz/tests/test_3d.py

```diff
@@ -238,6 +257,17 @@ def test_plot_alignment():
                    coord_frame='mri', subjects_dir=subjects_dir,
                    surfaces=['brain'], bem=sphere, show_axes=True)
 
+    # 3D coil with no defined draw (ConvexHull)
+    info_cube = pick_info(info, [0])
+    info_cube['chs'][0]['coil_type'] = 9999
+    with pytest.raises(RuntimeError, match='coil definition not found'):
+        plot_alignment(info_cube, meg='sensors', surfaces=())
+    coil_def_fname = op.join(str(tmpdir), 'temp')
```

Comment:
If you are using pytest tmpdir I would do this:
`coil_def_fname = tmpdir.join('temp')`